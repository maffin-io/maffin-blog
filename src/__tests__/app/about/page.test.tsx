import { render } from '@testing-library/react';

import AboutPage from '@/app/about/page';

describe('AboutPage', () => {
  it('renders as expected', () => {
    const { container } = render(<AboutPage />);

    expect(container).toMatchSnapshot();
  });
});
