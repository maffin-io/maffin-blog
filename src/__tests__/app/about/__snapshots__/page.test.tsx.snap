// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage renders as expected 1`] = `
<div>
  <div
    class="divide-y divide-gray-200 dark:divide-gray-700"
  >
    <div
      class="space-y-2 pt-6 pb-8 md:space-y-5"
    >
      <h1
        class="text-3xl font-extrabold leading-9 tracking-tight text-gray-900 dark:text-gray-100 sm:text-4xl sm:leading-10 md:text-6xl md:leading-14"
      >
        About
      </h1>
    </div>
    <div
      class="items-start space-y-2 xl:grid xl:grid-cols-3 xl:gap-x-8 xl:space-y-0"
    >
      <div
        class="flex flex-col items-center pt-8"
      >
        <img
          alt="Logo"
          class="h-48 w-48"
          data-nimg="1"
          decoding="async"
          height="192"
          loading="lazy"
          src="/_next/image?url=%2Fimg.jpg&w=384&q=75"
          srcset="/_next/image?url=%2Fimg.jpg&w=256&q=75 1x, /_next/image?url=%2Fimg.jpg&w=384&q=75 2x"
          style="color: transparent;"
          width="192"
        />
        <h3
          class="pt-4 pb-2 text-2xl font-bold leading-8 tracking-tight"
        >
          Maffin
        </h3>
        <div
          class="text-gray-500 dark:text-gray-400"
        >
          Personal finance management
        </div>
        <div
          class="italic text-gray-500 dark:text-gray-400"
        >
          made easy
        </div>
      </div>
      <div
        class="prose max-w-none pt-8 pb-8 dark:prose-dark xl:col-span-2"
      >
        Maffins mission is to make personal finance easy. Loads out there manage their finances using their custom spreadsheets or other money management tools and sometimes it just works but, the moment you start adding multiple currencies, investments, etc. it complicates it a lot.
        

        Maffin differentiators are:
        

        <ul>
          <li>
            <b>
              Data is yours
            </b>
            . We dont host any of your data, you store your own data and we just interpret it in the browser, it never reaches our servers.
          </li>
          <li>
            <b>
              Transparency is key
            </b>
            . We know how private personal finance is and to prove we dont do anything weird, all our code is open source (in progress due to licensing). We love contributing back.
          </li>
          <li>
            <b>
              Data platform
            </b>
            . Our main goal is to provide people a way to store their personal finances and visualize them. Thats it. We are not going to sell you insurances or recommend investment products. Our money comes from the hosted platform and donations/sponsors (through opensource).
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
